### ISSUE TITLE
Error: Cannot find module '@nodelib/fs.stat'

### ISSUE THREAD

**Issue by stolinski** [ardatan/graphql-toolkit#352]
I'm getting an error when building on http://zeit.co/

```
Error: Cannot find module '@nodelib/fs.stat'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/var/task/node_modules/fast-glob/out/readers/stream.js:4:16)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/var/task/node_modules/fast-glob/out/providers/async.js:3:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32) code: 'MODULE_NOT_FOUND' }
```

After running `npm ls @nodelib/fs.stat`

I could see that is leads to fast-glob which this library is using

```
└─┬ graphql-toolkit@0.7.4
  └─┬ @graphql-toolkit/core@0.7.4
    └─┬ globby@10.0.1
      └─┬ fast-glob@3.1.0
        ├── @nodelib/fs.stat@2.0.3
        └─┬ @nodelib/fs.walk@1.2.4
          └─┬ @nodelib/fs.scandir@2.1.3
            └── @nodelib/fs.stat@2.0.3  deduped
```

Anyone else seeing this?

**Comment by ardatan** [comment]
I think it is an issue related to `@zeit/now`. 
See;
https://github.com/zeit/node-file-trace/issues/55
https://github.com/acornjs/acorn-stage3/issues/4

**Comment by stolinski** [comment]
I'm using it via https://github.com/accounts-js/accounts so not as a direct dep, but something this library is relying on. Hopefully they can get the acorn issue worked out.

**Comment by TAnas0** [comment]
I am facing the same issue trying to build a minimal Apollo Server deployable on Now.

I am depending on graphql-toolkit because I am using GraphQL Modules. Here is the output of `npm ls @nodelib/fs.stat`:
```
└─┬ @graphql-modules/core@0.7.13
  └─┬ graphql-toolkit@0.5.16
    └─┬ globby@10.0.1
      └─┬ fast-glob@3.1.1
        ├── @nodelib/fs.stat@2.0.3 
        └─┬ @nodelib/fs.walk@1.2.4
          └─┬ @nodelib/fs.scandir@2.1.3
            └── @nodelib/fs.stat@2.0.3  deduped
```

I don't know if you are still working on this issue, but I have a reproducible example in here. Use this to get the exact commit: `git checkout de2b99a`

More importantly, is this issue solved by switching to the `@graphql-toolkit/*` monorepos? If so, for the life of me I cannot find them or any help in the Readme. Could you point me to where to get them?

@ardatan I have been going down this rabbit hole, and to my understanding, this issue is in fact more related to the Now builder than it is related to this repository, so this issue might be out of your hand.

**Comment by ardatan** [comment]
@TAnas0 I released 0.7.14 that uses graphql-toolkit monorepos. Could you try it and let me know if the problem persists?
